packages/rich-text/src/to-dom.js

```diff
@@ -110,6 +105,33 @@ function remove( node ) {
 	return node.parentNode.removeChild( node );
 }
 
+function createBogusBR( doc ) {
+	const br = doc.createElement( 'br' );
+	br.setAttribute( 'data-mce-bogus', '1' );
+	return br;
+}
+
+function pad( node ) {
```

Comment:
The function is a little hard to decipher. Perhaps defining some terminology if terminology is needed (e.g. padding), and making the high-level purpose evident.